test_runner/src/main/kotlin/ftl/run/platform/android/ResolveApks.kt

```diff
@@ -21,7 +21,7 @@ internal fun AndroidArgs.resolveApks(): List<AndroidTestContext> = listOfNotNull
 
 private fun AndroidArgs.mainApkContext() = appApk?.let { appApk ->
     when {
-        testApk != null -> InstrumentationTestContext(app = appApk.asFileReference(), test = testApk.asFileReference(), environmentVariables = emptyMap())
+        testApk != null -> InstrumentationTestContext(app = appApk.asFileReference(), test = testApk.asFileReference(), environmentVariables = emptyMap(), testTargetsForShard = testTargetsForShard)
```

Comment:
This line is way too long. Please wrap lines longer than 80 columns, cause are unreadable on split screen.

Reply:
i wonder why detekt didnt pick it up